src/pages/Lend/AlgoliaPaginationWrapper.vue

```diff
@@ -1,14 +1,14 @@
 <template>
 	<ais-pagination :padding="padding">
 		<div slot-scope="{
-				currentRefinement,
-				nbPages,
-				pages,
-				isFirstPage,
-				isLastPage,
-				refine,
-				createURL
-			}"
+			currentRefinement,
+			nbPages,
+			pages,
+			isFirstPage,
+			isLastPage,
+			refine,
+			createURL
+		}"
 		>
```

Comment:
I think this is probably the worst consequence of having the dangling closing bracket.